releases/release-5.0.6.md

```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 5.0.6 Release Notes
+---
+
+# TiDB 5.0.6 Release Notes
+
+发版日期：2021 年 12 月 28 日
+
+TiDB 版本：5.0.6
+
+## 兼容性更改
+
++ Tools
+
+    + TiCDC
+
+        - cdc server 命令错误输出从标准输出改成标准错误. [#3133](https://github.com/pingcap/tiflow/issues/3133)
+
+## 提升改进
+
++ TiDB
+
+    (dup) - 修复乐观事务冲突可能导致事务相互阻塞的问题 [#11148](https://github.com/tikv/tikv/issues/11148)
+    (dup) - 修复运行 MPP 查询时出现 `invalid cop task execution summaries length` 相关日志的问题 [#1791](https://github.com/pingcap/tics/issues/1791)
+    (dup) - 当 coprocessor 遇到锁时，在调试日志中显示受影响的 SQL 语句帮助诊断问题 [#27718](https://github.com/pingcap/tidb/issues/27718)
+
++ TiKV
+
+    - 将插入 SST 文件时的校验操作移动到了 import 线程池以提高 SST 文件插入速度. [#11239](https://github.com/tikv/tikv/issues/11239)
+    - 为 raft 日志回收模块增加了更多监控以定位问题. [#11374](https://github.com/tikv/tikv/issues/11374)
+    - 折叠了部分 Storage 相关的不常用监控 [#11681](https://github.com/tikv/tikv/issues/11681)
+
++ PD
+
+    (dup) - 优化调度器退出的速度 [#4146](https://github.com/tikv/pd/issues/4146)
+    - 通过允许空 Region 调度和修复配置的方式使 scatter-range-scheduler运行更好 [#4497](https://github.com/tikv/pd/issues/4497)
+
++ Tools
+
+    + TiCDC
+
+        (dup) - 优化 TiKV 重新加载时的速率限制控制，缓解 changefeed 初始化时 gPRC 的拥堵问题 [#3110](https://github.com/pingcap/tiflow/issues/3110)
+        (dup) - 修复当扫描存量数据耗时过长时，可能由于 TiKV 进行 GC 而导致存量数据扫描失败的问题 [#2470](https://github.com/pingcap/tiflow/issues/2470)
+        (dup) - 修复当发生 ErrGCTTLExceeded 错误时，changefeed 不快速失败的问题 [#3111](https://github.com/pingcap/tiflow/issues/3111)
+        (dup) - 为 EtcdWorker 添加 tick 频率限制，防止 PD 的 etcd 写入次数过于频繁影响 PD 服务 [#3112](https://github.com/pingcap/tiflow/issues/3112)
+        - 支持消息 Batch 操作减少 EtcdWorker tick [3112](https://github.com/pingcap/tiflow/issues/3112)
+        (dup) - 修复在容器环境中 OOM 的问题 [#1798](https://github.com/pingcap/tiflow/issues/1798)
+        - Kafka sink 模块支持默认的元数据获取超时时间 config.Metadata.Timeout [#3352](https://github.com/pingcap/tiflow/issues/3352)
+        - Kafka sink 模块设置 `MaxMessageBytes` 默认值为 1MB [#3081](https://github.com/pingcap/tiflow/issues/3081)
```

Comment:
```
        - Kafka sink 模块的 `MaxMessageBytes` 默认值设置为 10MB，并使用 Kafka broker 参数 `message.max.bytes` 和 Topic 参数 `max.message.bytes` 的较小值对 producer 进行初始化[#3081](https://github.com/pingcap/tiflow/issues/3081)
```

Reply:
> ```
>         - Kafka sink 模块的 `MaxMessageBytes` 默认值设置为 10MB，并使用 Kafka broker 参数 `message.max.bytes` 和 Topic 参数 `max.message.bytes` 的较小值对 producer 进行初始化[#3081](https://github.com/pingcap/tiflow/issues/3081)
>         - 删除 TiCDC 内部 owner 和 processor 重构后遗留的相关历史代码 [#2765](https://github.com/pingcap/tiflow/issues/2765)
> ```

The notes should point out the benefits to users, instead of describing what we have done to the codes. Please optimize the notes. Thanks.

Reply:
Does this involve configuration value change? If yes, please add it to the **Compatibility changes** part. Thanks!
@shichun-0415 @maxshuang

Reply:
I have moved the configuration value change to Compatibility changes part.